package.json

```diff
@@ -34,7 +34,7 @@
     "clipboard": "^2.0.4",
     "connect": "^3.6.6",
     "eslint": "^5.16.0",
-    "govuk-frontend": "^3.7.0",
+    "govuk-frontend": "github:alphagov/govuk-frontend#86b284a6",
```

Comment:
Just need to remove this reference, after which I think this PR will be good to go.